tests/validationHelpers.js

```diff
@@ -0,0 +1,35 @@
+const locations = require('../dist/locations.json');
+const timeseries = require('../dist/timeseries-byLocation.json');
```

Comment:
@lazd @qgolsteyn should I pull from `coronadatascraper.com/*.json` instead of from `dist/` ?

Reply:
I would say there isn't much use in testing coronadatascraper.com/*.json as this data would have been released already.

Reply:
so the idea was to run this on our existing data and not any data produced during a dev run of a scrape. Perhaps a better/clearer name for it would be `dataQualityChecker` ? I think the missing piece is a summary that can report the quality of each dataset we have based on these test criteria. Did you or @lazd have a different idea in mind?